ast/json.go

```diff
@@ -776,6 +776,76 @@ func (e *ArrayExpression) UnmarshalJSON(data []byte) error {
 	}
 	return nil
 }
+func (e *DictExpression) MarshalJSON() ([]byte, error) {
```

Comment:
I am remembering that we define these JSON structures in the swagger doc for InfluxDB. Can you create an issue to update those definitions for these new AST nodes?

Reply:
https://github.com/influxdata/influxdb/issues/20299